docs/connection-options.md

```diff
@@ -185,6 +185,8 @@ See [SSL options](https://github.com/mysqljs/mysql#ssl-options).
 
 * `logNotifications` - A boolean to determine whether postgres server [notice messages](https://www.postgresql.org/docs/current/plpgsql-errors-and-messages.html) and [notification events](https://www.postgresql.org/docs/current/sql-notify.html) should be included in client's logs with `info` level (default: `false`).
 
+* `skipInstallExtensions` - A boolean to control whether to install necessary postgres extensions automatically or not (default: `true`)
```

Comment:
The documented default is incorrect.

Also, why not just make this `installExtensions` instead of negating it?

Reply:
> The documented default is incorrect.

True I overlooked this.


> Also, why not just make this installExtensions instead of negating it?

I initially did it that way (`installExtensions`) but I couldn't figure out how to make it `true` by default so I opted for `skipInstallExtensions` which will be `false` by default. We don't want to change the existing behavior

Reply:
I would prefer to rename it `installExtensions` if you can lead me to how I can make it default to `true`

Reply:
@imnotjames In any case, I've updated this documentation to make it correct for this case